lib/user.rb

```diff
@@ -0,0 +1,64 @@
+require "./lib/database_connection"
+require "./lib/select_database"
+require "bcrypt"
+
+class User
+
+  attr_reader :id, :email, :username, :name, :password
```

Comment:
I don't think you want to make password accessible outside the user object. I don't think you're using it anywhere for a start (I don't see any authentication) but usually we would want to keep this private and have an `authenticate` method which validates a users password without exposing the information.

In general, try not to expose information until you need it - attr_readers should only be added where other classes actually need to read state, rather than automatically when you create a new class.